FirebaseCore/CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
-# Unreleased
-  [changed] Directory structure reorganization without functional changes.
+# v6.6.0 -- M62
+- [changed] Directory structure reorganization without functional changes.
```

Comment:
This can be rephrased to `Reorganized directory structure.` We should also consider omitting this from devsite (but not from GitHub) since it results in no binary changes.

Reply:
I like the rephrase but not the omission. Our product is source not binary - and if someone is doing something weird with the directory structure, this will impact them.

Reply:
rephrased to `Reorganized directory structure.`